lib/event_listeners/addFood.js

```diff
@@ -1,21 +1,31 @@
 const $ = require('jquery')
+const Food = require('../models/Food')
+const Ajax = require('../ajax_requests/ajax')
 
 $(document).ready(function(){
```

Comment:
One thing we will have to add to the ToDo list is to refactor our code all into es5 or es6 before completion